src/modules/billing/jobs/update-customer.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const { get, partial } = require('lodash');
+const chargeModuleCustomersConnector = require('../../../lib/connectors/charge-module/customers');
+const JOB_NAME = 'billing.update-customer-account';
+const invoiceAccountsConnector = require('../../../lib/connectors/crm-v2/invoice-accounts');
+const chargeModuleMappers = require('../../../lib/mappers/charge-module');
+const { logger } = require('../../../logger');
+const uuid = require('uuid/v4');
+
+const messageInitialiser = (jobName, invoiceAccountId) => ([
+  JOB_NAME,
+  {
+    invoiceAccountId
+  },
+  {
+    singletonKey: `${JOB_NAME}.${invoiceAccountId}.${uuid()}`,
+    retryLimit: 3,
+    retryDelay: 30,
+    retryBackoff: true
+  }
+]);
+
+const createMessage = partial(messageInitialiser, JOB_NAME);
+
+const handler = async job => {
+  const invoiceAccountId = get(job, 'data.invoiceAccountId');
+  const invoiceAccountData = await invoiceAccountsConnector.getInvoiceAccountById(invoiceAccountId);
```

Comment:
I think you should use the invoice accounts service here (`src/lib/services/invoice-accounts-service.js`). It will return the data already mapped to the model, so you won't have to map it to the model in the `mapInvoiceAccountToChargeModuleCustomer`

Reply:
That's a great suggestion. I'll implement it. Thanks :)